tools/pihole_toml_to_markdown.py

```diff
@@ -0,0 +1,174 @@
+import re
+from pathlib import Path
+
+def parse_toml_with_comments(filepath):
+    with open(filepath, encoding='utf-8') as f:
+        lines = f.readlines()
+
+    documentation = ["""---
+description: This file is automatically generated when new code is pushed to the master branch of the FTL Repo. Any proposed changes should be made against https://github.com/pi-hole/ftl/tools/pihole_toml_to_markdown.py
+---
+
+# Pi-hole FTL Configuration Reference
+
+This page documents the available options in the `pihole-FTL.conf` file, which is typically managed via the TOML-formatted configuration file:
+
+```text\n/etc/pihole/pihole.toml\n```
+
+The file can be edited directly, however you can also use the command line option or the web interface.
+
+To edit with the command line, use the format `key.name=value`, e.g:
+
+```text\npihole-FTL --config dns.dnssec=true\n```
```

Comment:
```suggestion
```text\nsudo pihole-FTL --config dns.dnssec=true\n```
```
`sudo` will be needed unless you are either user `pihole` (or `root`)

Reply:
Cool, I'll do this one in a separate commit, because there is another place